dpgen/auto_test/Interstitial.py

```diff
@@ -147,9 +148,10 @@ def make_confs(self,
                     os.remove(insert_element_task)
 
                 for ii in self.insert_ele:
-                    vds = InterstitialGenerator(ss, ii)
+                    pre_vds = InterstitialGenerator()
+                    vds = pre_vds.generate(ss, {self.insert_ele[0]: [[0, 0.2, 0.5]]})
```

Comment:
Only the first element of the `insert_ele` is inserted to the system. Its it true?  I do not how other elements in `insert_ele` is used in the new code.

Reply:
[0, 0.2, 0.5] is hard coded. Could you please explain this parameter? In any case a user wants to change it?

Reply:
It is the coordinates of the initial insertion. Since the insertions of the previous version of the Pymagen were not well defined, the function of this part of the codes was accomplished by random insertion of an atom into the bulk and then revising its position directly on the POSCAR. It was a compromise approach until the new Pymatgen come out where the position and type of the inserted atom can be specified easily. Thus, this PR just aims to make current codes compatible with the new Pymatgen, and we will rewrite this py later to make it more robust.

Reply:
It seems that the old behavior of interstitial generation is no longer supported in the latest pymatgen-analyze-defect, is it true?

Reply:
YES, all defect-related functions in the Pymatgen after v2022.7.19 are transferred to the pymatgen-analysis-defects plugin. Additionally, this part was rewritten and some of the old functions and class methods are no longer valid.

Reply:
Then what is the plan of pymatgen? Currently you provide the type `self.insert_ele[0]` and the coordinate `[[0, 0.2, 0.5]]` of the interstitial atom. Notice that the old behavior is to exhausitvely search all possible interstitial sites.

Reply:
I have changed self.insert_ele[0] into ii in self.insert_ele so that it can iterate over the list. As for the coordinate [0, 0.2, 0.5], since the new version of pymatgen no longer provides the function to search interstitial sites automatically, I offer an initial input coordinate [0.1, 0.1, 0.1] which won't possibly conflict with any lattice atom to kick off the code.
If this plan is still not good enough, I will rewrite the current make_confs method to be better compatible with the new pymatgen

Reply:
It would be better to directly ask the pymatgen people how to reproduce the old behavior with the new code.

Providing the coordinate like `[0.1, 0.1, 0.1] ` only gives you the  interstitial site nearest to `[0.1, 0.1, 0.1] ` , am I correct?

Reply:
Thank you very much for the explanation. I accept the changes in this PR.